index.html

```diff
@@ -14,36 +15,2214 @@
 <body>
   <div class="container">
     <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+      <h1>Week 1: Unemployment Rates Since 1980</h1>
+      <p>This table shows two unemployment rates measured every month since 1980.
+	  The second column is the civilian unemployment rate. The third column is the percentage out of total unemployed
+	  who have been unemployed for 27 weeks or more.
+	  <br>
+	  <br>
+	  
     </div>
-    <table class="table">
+    <center><table class="table">
```

Comment:
⚠️ The CENTER tag has been deprecated, and we shouldn't use it. For good reason, I should add: This is declaring a style choice in our HTML markup. We'd prefer our markup to be focused on content, while our CSS holds anything related to the visual presentation. We could achieve the same effect with CSS with `.table { margin: 0 auto; }`, which is shorthand for setting the top and bottom margins (distance between this element and other elements on the page) to 0, and the left and right margins to be automatically calculated (effectively centering the element on the page).